--- t-w.conf	Sat Mar 18 17:04:05 2006
+++ toaster-watcher.conf-dist	Sat Mar 18 16:12:53 2006
@@ -40,11 +40,11 @@
 toaster_prefix                 = /usr/local
 toaster_src_dir                = /usr/local/src
 toaster_debug                  = 0
-toaster_hostname               = jail10.cadillac.net
+toaster_hostname               = mail.example.com
 system_config_dir              = /usr/local/etc
-toaster_admin_email            = postmaster@jail10.cadillac.net
-toaster_test_email             = test@jail10.cadillac.net
-toaster_test_email_pass        = cHanGeMed
+toaster_admin_email            = postmaster@example.com
+toaster_test_email             = test@example.com
+toaster_test_email_pass        = cHanGeMe
 mail_syslog                    = /var/log/maillog
 package_install_method         = ports  # packages | ports
                                  # if both options are supported, the setup
@@ -68,15 +68,15 @@
 # and will install trying first via ports, then sources if the port install fails.
 
 install_squirrelmail           = port   # 0, ver, port (1.4.3a)
-install_squirrelmail_sql       = 1      # use MySQL for user prefs and address book?
+install_squirrelmail_sql       = 0      # use MySQL for user prefs and address book?
 install_apache                 = 2      # 0, 1, 2, 21, 22
 install_apache_etc             = 0      # defaults to /usr/local/etc/apache[2]
 install_apache_user            = www
-install_apache2_modperl        = 1      # 0, 1, ver
+install_apache2_modperl        = 0      # 0, 1, ver
 install_apache_suexec          = 0      # 0, 1
 install_apache_proxy           = 0      # 0, 1
 install_php                    = 4      # 0, 4, 5, ver
-install_phpmyadmin             = 1      # 0, 1
+install_phpmyadmin             = 0      # 0, 1
 install_mysql                  = 41     # 0, 1, 2, 3, 40, 41, 5
 install_mysql_ssl              = 1
 install_mysql_linuxthreads     = 0
@@ -88,15 +88,15 @@
 install_netqmail               = 1.05   # 0, ver       (1.05)
 install_qmailadmin             = 1.2.9  # 0, ver, port (1.2.9)
 install_vpopmail               = 5.4.10 # ver, port    (5.4.7)
-install_vqadmin                = 1      # 0, ver, port
+install_vqadmin                = 0      # 0, ver, port
 install_isoqlog                = port   # 0, ver, port (2.2)
 install_portupgrade            = 1
 install_portaudit              = 1
-install_openldap_client        = 1
+install_openldap_client        = 0
 install_net_snmpd              = 4      # 4, 5
 install_rrdutil                = 1
 install_ezmlm                  = 0.42   # 0, ver
-install_ezmlm_cgi              = 1      # 0, 1
+install_ezmlm_cgi              = 0      # 0, 1
 install_openssl_port           = 1
 install_qmail_notify           = 0
 install_aspell                 = 0
@@ -115,16 +115,16 @@
 install_spamassassin_sql       = 0       # use AWL, bayes, and per-user prefs from MySQL
 install_spamassassin_dbuser    = spamassassin
 install_spamassassin_dbpass    = assSPAMing
-install_dspam                  = 1
+install_dspam                  = 0
 install_simscan                = 1.2     # 0, ver
-install_qmailscanner           = 1.25    # 0, ver       (1.25  as of 04/08/05)
+install_qmailscanner           = 0       # 0, ver       (1.25  as of 04/08/05)
 install_qmailscanner_stats     = 0       # 0, ver       (2.0.2 as of 02/02/05)
 install_clamav                 = port    # 0, port, ver (0.84  as of 05/02/05)
 install_clamav_user            = clamav  # qscand | clamav
-install_pyzor                  = 1
+install_pyzor                  = 0
 install_razor                  = port    # 0, port, ver (2.80  as of 03/15/06)
-install_bogofilter             = 1
-install_dcc                    = 1
+install_bogofilter             = 0
+install_dcc                    = 0
 install_ripmime                = port    # 0, port, ver (1.4.0.6 as of 3/15/06)
 
 filtering_method                   = tcpserver  # smtp | tcpserver  # see docs for details
@@ -181,7 +181,7 @@
 vpopmail_group                 = vchkpw
 vpopmail_home_dir              = /usr/local/vpopmail
 vpopmail_learn_passwords       = 1
-vpopmail_default_domain        = jail10.cadillac.net
+vpopmail_default_domain        = 0
 vpopmail_roaming_users         = 1
 vpopmail_relay_clear_minutes   = 180
 vpopmail_mysql                 = 1
@@ -363,7 +363,7 @@
 #            POP3D-SSL                #
 #######################################
 
-pop3_ssl_daemon                = qpop3d  #  qpop3d | courier
+pop3_ssl_daemon                = courier #  qpop3d | courier
 
 # NOTICE: if you choose qpop3d, then stunnel will be installed
 # and courier's pop3ssl will be disabled.
@@ -425,7 +425,7 @@
 qmailadmin_install_as_root      = 0
 qmailadmin_modify_quotas        = 1
 qmailadmin_domain_autofill      = 1
-qmailadmin_return_to_mailhome   = 1
+qmailadmin_return_to_mailhome   = 0
 qmailadmin_spam_command         = | /usr/local/bin/maildrop /usr/local/etc/mail/mailfilter
 qmailadmin_cgi-bin_dir          = 0  # override toaster_cgi-bin
 qmailadmin_http_docroot         = 0  # override toaster_http_docs
