--- toaster-watcher.conf-dist	2006-03-10 10:23:46.000000000 -0600
+++ toaster-watcher.conf	2006-03-10 11:12:34.000000000 -0600
@@ -22,7 +22,7 @@
 #            TOASTER
 ######################################
 
-freebsd_os_release             = 5-stable
+freebsd_os_release             = darwin
 cvsup_server_preferred         = fastest         # fastest or a hostname
 cvsup_server_country           = US
 cvsup_supfile_ports            = /etc/cvsup-ports
@@ -33,15 +33,15 @@
 toaster_dl_site                = http://www.tnpi.biz    # select a mirror
 toaster_dl_url                 = /internet/mail/toaster
 toaster_version                = 0     # 0 = latest release # v = 4.01
-toaster_http_base              = /usr/local/www
-toaster_http_docs              = /usr/local/www/data
-toaster_cgi-bin                = /usr/local/www/cgi-bin
+toaster_http_base              = /Library/WebServer
+toaster_http_docs              = /Library/WebServer/Documents
+toaster_cgi-bin                = /Library/WebServer/CGI-Executables
 toaster_tmp_dir                = /tmp
-toaster_prefix                 = /usr/local
-toaster_src_dir                = /usr/local/src
+toaster_prefix                 = /opt/local
+toaster_src_dir                = /opt/local/src
 toaster_debug                  = 0
 toaster_hostname               = mail.example.com
-system_config_dir              = /usr/local/etc
+system_config_dir              = /opt/local/etc
 toaster_admin_email            = postmaster@example.com
 toaster_test_email             = test@example.com
 toaster_test_email_pass        = cHanGeMe
@@ -67,7 +67,7 @@
 # Extra options are noted after the # where available. Selecting 1 is the most reliable
 # and will install trying first via ports, then sources if the port install fails.
 
-install_squirrelmail           = port   # 0, ver, port (1.4.3a)
+install_squirrelmail           = 1.4.6  # 0, ver, port (1.4.4)
 install_squirrelmail_sql       = 0      # use MySQL for user prefs and address book?
 install_apache                 = 2      # 0, 1, 2, 21, 22
 install_apache_etc             = 0      # defaults to /usr/local/etc/apache[2]
@@ -77,7 +77,7 @@
 install_apache_proxy           = 0      # 0, 1
 install_php                    = 4      # 0, 4, 5, ver
 install_phpmyadmin             = 1      # 0, 1
-install_mysql                  = 40     # 0, 1, 2, 3, 40, 41, 5
+install_mysql                  = 0      # 0, 1, 2, 3, 40, 41, 5
 install_mysql_ssl              = 1
 install_mysql_linuxthreads     = 0
 install_mysql_optimized        = 1
@@ -89,8 +89,8 @@
 install_qmailadmin             = 1.2.7  # 0, ver, port (1.2.7)
 install_vpopmail               = 5.4.10 # ver, port    (5.4.7)
 install_vqadmin                = 0      # 0, ver, port
-install_isoqlog                = port   # 0, ver, port (2.2)
-install_portupgrade            = 1
+install_isoqlog                = 2.2    # 0, ver, port (2.2)
+install_portupgrade            = 0
 install_openldap_client        = 1
 install_net_snmpd              = 4      # 4, 5
 install_rrdutil                = 1
@@ -128,7 +128,7 @@
 
 filtering_method                   = tcpserver  # smtp | tcpserver  # see docs for details
 filtering_spamassassin_method      = site       # site | user | domain
-filtering_maildrop_filter_file     = /usr/local/etc/mail/mailfilter
+filtering_maildrop_filter_file     = /opt/local/etc/mail/mailfilter
 filtering_spama_discard_score      = 12         # discard messages with a SA score above this
 filtering_report_spam_spamassassin = 1
 filtering_report_spam_pyzor        = 0          # don't enable this with report_spamassassin
@@ -161,7 +161,7 @@
 qmail_log_base                 = /var/log/mail
 qmail_log_user                 = qmaill
 qmail_log_group                = qnofiles
-qmail_mysql_include            = /usr/local/lib/mysql/libmysqlclient.a
+qmail_mysql_include            = /opt/local/lib/mysql/libmysqlclient.a
 qmail_group                    = qmail
 qmail_user_alias               = alias
 qmail_user_daemon              = qmaild
@@ -178,12 +178,12 @@
 
 vpopmail_user                  = vpopmail
 vpopmail_group                 = vchkpw
-vpopmail_home_dir              = /usr/local/vpopmail
+vpopmail_home_dir              = /opt/local/vpopmail
 vpopmail_learn_passwords       = 1
 vpopmail_default_domain        = 0
 vpopmail_roaming_users         = 1
 vpopmail_relay_clear_minutes   = 180
-vpopmail_mysql                 = 1
+vpopmail_mysql                 = 0
 vpopmail_mysql_limits          = 0
 vpopmail_mysql_replication     = 0
 vpopmail_mysql_logging         = 0
@@ -238,7 +238,7 @@
 smtpd_max_memory_per_connection = 15            # in megabytes
 smtpd_max_connections           = 20
 smtpd_max_memory                = 356
-smtpd_use_mysql_relay_table     = 1
+smtpd_use_mysql_relay_table     = 0
 smtpd_lookup_tcpremotehost      = 0
 smtpd_lookup_tcpremoteinfo      = 0
 smtpd_dns_paranoia              = 0
@@ -425,10 +425,10 @@
 qmailadmin_modify_quotas        = 1
 qmailadmin_domain_autofill      = 1
 qmailadmin_return_to_mailhome   = 0
-qmailadmin_spam_command         = | /usr/local/bin/maildrop /usr/local/etc/mail/mailfilter
+qmailadmin_spam_command         = | /opt/local/bin/maildrop /opt/local/etc/mail/mailfilter
 qmailadmin_cgi-bin_dir          = 0  # override toaster_cgi-bin
 qmailadmin_http_docroot         = 0  # override toaster_http_docs
-qmailadmin_http_images          = /usr/local/www/data/images
+qmailadmin_http_images          = /Library/WebServer/Documents/images
 
 
 #######################################
@@ -526,9 +526,9 @@
 #      Apache Suexec Options          #
 #######################################
 
-apache_suexec_docroot          = /usr/local/www/data
+apache_suexec_docroot          = /Library/WebServer/Documents
 apache_suexec_userdir          = html
-apache_suexec_safepath         = /usr/local/bin:/usr/bin:/bin
+apache_suexec_safepath         = /opt/local/bin:/usr/bin:/bin
 apache_suexec_logfile          = /var/log/apache/suexec.log
 apache_suexec_uidmin           = 1000
 apache_suexec_gidmin           = 1000
